installation/scripts/testing-auditlog-temp.sh

```diff
@@ -28,6 +28,8 @@ matchNames=$(cat <<-END
         namespace: compass-system
       - name: compass-director-gateway-integration
         namespace: kyma-system
+      - name: compass-external-services-mock
+        namespace: kyma-system
```

Comment:
why not compass-system? Just curious

Reply:
Because the test needs access to dex credentials which are stored in secret in `kyma-system` namespace